src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -208,25 +208,26 @@ String getAppPath(String appName) {
      */
     @Override
     String processClassPath(String classPath) throws IOException {
-        if (!this.projectConfiguration.isUseJavaFX()) {
+        if (!projectConfiguration.isUseJavaFX()) {
             return classPath;
         }
         // we are using JavaFX
-        String javafxSDK = FileDeps.getJavaFXSDK(projectConfiguration).resolve("lib").toString();
+        String javafxSDKLibs = FileDeps.getJavaFXSDKLibsPath(projectConfiguration).toString();
 
-        StringBuffer answer = new StringBuffer();
-        Stream.of(classPath.split(File.pathSeparator)).forEach(s ->{
-            if (s.indexOf("javafx") < 0 ){
-                answer.append(s).append(File.pathSeparator);
+        StringBuilder answer = new StringBuilder();
+        Stream.of(classPath.split(File.pathSeparator)).forEach(s -> {
```

Comment:
If you're using stream, then IMO it is better to use functional flow, i.e. map each element to appropriate string and then collect to string using path separator.

Reply:
ok, refactored